config/site/site.config.js

```diff
@@ -123,6 +123,16 @@ const siteConfig = {
         gitHubUrl: npmPackage.repository.url.replace('git+', ''),
       },
     },
+
+
+    /** This section allows to you inject arbitrary html into the head tag of index.html.
+     *  It takes an array of strings. You could load the string from an external file if desired.
+     *  Add like:
+     *  headHtml:[
+     *    '<script> console.log("Terra Dev Site!") </script>',
+     *  ],
+     */
+    headHtml: [],
```

Comment:
Are there any looping concerns if someone extends the default and this value ends up undefined?

Reply:
They'd have to go out of their way to break it. I'm not too concerned. We make the same assumption with other values in our default config.